internal/cmd/asyncapi/command_export.go

```diff
@@ -25,15 +27,26 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/log"
 	"github.com/confluentinc/cli/internal/pkg/output"
 	"github.com/confluentinc/cli/internal/pkg/serdes"
+	"github.com/confluentinc/cli/internal/pkg/types"
 )
 
 type command struct {
 	*pcmd.AuthenticatedCLICommand
 }
 
 type confluentBinding struct {
-	XPartitions int32             `json:"x-partitions,omitempty"`
-	XConfigs    map[string]string `json:"x-configs,omitempty"`
+	BindingVersion string            `json:"bindingVersion,omitempty"`
+	Partitions     int32             `json:"partitions,omitempty"`
+	TopicConfigs   topicConfig       `json:"topicConfiguration,omitempty"`
```

Comment:
Let's match with the JSON name for clarity
```suggestion
	TopicConfiguration   topicConfiguration       `json:"topicConfiguration,omitempty"`
```